src/content/release-notes.mdx

```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.24
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded ingress-nginx to v4.11.2  <!-- 8460 -->
+
+### New Features
+
+- Add provider credentials query for deploy and test executions  <!-- https://github.com/okteto/app/pull/8454 -->
+- Enable [multi-user token authentication](self-hosted/install/auth/token.mdx)  <!-- https://github.com/okteto/app/pull/8481 -->
+- Announcing Okteto CLI 3.0 [Will link to blog post]
+
+### Improvements
+
+- Added new Developer Quickstart Guide to the documentation <!-- to add link on version upgrade -->
+- Updated the "Want to know more" links in an empty namespace  <!-- 8415 -->
+- Added tolerations for EventsExporter StatefulSet  <!-- 8446 -->
```

Comment:
This is a feature, not an improvement. @jvc5546 Could you document the event exporter section in our helm values?

Reply:
@codyjlandstrom added a PR to document the event exporter. I would link to the event exporter docs here:
https://github.com/okteto/docs/pull/821